packages/lib/src/spatial-navigation/components/ScrollView.tsx

```diff
@@ -59,9 +160,63 @@ export const SpatialNavigationScrollView = ({
         showsHorizontalScrollIndicator={false}
         showsVerticalScrollIndicator={false}
         scrollEnabled={false}
+        onScroll={(event) => {
+          scrollY.current = event.nativeEvent.contentOffset.y;
+        }}
+        scrollEventThrottle={16}
       >
         {children}
       </ScrollView>
+      {deviceType === 'remotePointer' ? (
+        <PointerScrollArrows
+          descendingArrow={descendingArrow}
+          ascendingArrow={ascendingArrow}
+          descendingArrowContainerStyle={descendingArrowContainerStyle}
+          ascendingArrowContainerStyle={ascendingArrowContainerStyle}
+          ascendingArrowProps={ascendingArrowProps}
+          descendingArrowProps={descendingArrowProps}
+        />
+      ) : undefined}
     </SpatialNavigatorParentScrollContext.Provider>
   );
 };
+
+const PointerScrollArrows = React.memo(
+  ({
+    ascendingArrow,
+    descendingArrowProps,
+    ascendingArrowContainerStyle,
+    descendingArrow,
+    ascendingArrowProps,
+    descendingArrowContainerStyle,
+  }: {
+    ascendingArrow?: ReactElement;
+    ascendingArrowProps?: {
+      onMouseEnter: () => void;
+      onMouseLeave: () => void;
+    };
+    ascendingArrowContainerStyle?: ViewStyle;
+    descendingArrow?: ReactNode;
+    descendingArrowProps?: {
+      onMouseEnter: () => void;
+      onMouseLeave: () => void;
+    };
+    descendingArrowContainerStyle?: ViewStyle;
+  }) => {
+    const DescendingArrowContainer = styled.View({
```

Comment:
why do you declare this component dynamically in the render function?
You should take it out of the function and declare it at root level, next to the PointerScrollArrows declaration!

Reply:
Done :)